### PR TITLE
Update architecture docs with ena-deposition state

### PR DESCRIPTION
**Pull request by anna-parker**
resolves #

preview URL:

### PR Checklist

- [ ] All necessary documentation has been adapted.
- [ ] The implemented feature is covered by an appropriate test.

### CODE REVIEW

**Review by fengelniederhammer — COMMENTED** [review]
Thanks for the work!

I have a couple of suggestions, but this helps a lot.

File: architecture_docs/plantuml/06_ena_deposition.puml [hunk 1]
```diff
@@ -2,13 +2,20 @@
...
-
-' What happens next?
+enacronjob -> backend: Query for all sequences
+backend -> enacronjob: All sequences
+enacronjob -> slack: All new sequences to deposit in ENA
+slack -> github: Manual upload
+github -> deposition: Upload of sequences to deposit in ENA
+deposition -> ena: Deposition of sequences in ENA
+ena -> deposition: Results of deposition (accessions)
+deposition -> backend: Upload of deposition results as external metadata
```
[the remark this answers is not in the document]
Comment (anna-parker) [reply architecture_docs/plantuml/06_ena_deposition.puml:19]: If upload of external metadata is not successful we get into a failed upload state in the db and we get a slack notification

[the remark this answers is not in the document]
Comment (anna-parker) [reply architecture_docs/plantuml/06_ena_deposition.puml]: In general we get into a bad state if a step in the ena submission fails - these currently all require manual intervention. The idea is to never submit a sequence/sample/project more than once.

File: architecture_docs/06_runtime_view.md [hunk 2]
```diff
@@ -24,4 +24,8 @@ If the data has been reprocessed successfully, they can approve it, and it will
...
+
+The ena-deposition pod queries this github repo for new sequences. Upon finding a new sequence to submit to ENA the pod uploads it to the `ena-deposition` schema in the backend. This triggers the ENA submission process, there are three main steps to this submission: creation of projects, samples and assemblies. If all these steps succeed the ena-deposition pod will upload the results of this submission (e.g. new INSDC accessions) as external metadata to the backend using the `submit-external-metadata` backend-endpoint. The backend only accepts metadata in this request that is in the `externalMetadata` schema. The backend then updates the metadata in the `sequence_entries_view` in the database with the external metadata values - leading to the results being shown on the website.
```
[the remark this answers is not in the document]
Comment (fengelniederhammer) [reply architecture_docs/06_runtime_view.md]: I moved it to the ENA submission README

**Review by fengelniederhammer — APPROVED** [review]

**Pull request merged**